client/model-builder/src/main/java/io/smallrye/graphql/client/model/helper/TypeModel.java

```diff
@@ -361,6 +365,23 @@ public boolean isTypeVariable() {
         return Classes.isTypeVariable(type);
     }
 
+    /**
+     * Checks if the type is a Union.
+     *
+     * @return {@code true} if the type is a Union, otherwise {@code false}.
+     */
```

Comment:
This javadoc explains nothing that isn't already in the method name. I'd rather delete it. Or link to the documentation what a GraphQL Union is: https://spec.graphql.org/draft/#sec-Unions

Reply:
```suggestion
    /**
     * @see <a href="https://spec.graphql.org/draft/#sec-Unions">GraphQL Union</a>
     */
```